config/clock.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'clockwork'
+require 'active_support/time'
+require './config/boot'
+require './config/environment'
+
+# TODO
+# Instead of loading whole application, we can use this approach of sending tasks to Sidekiq
+# https://github.com/mperham/sidekiq/wiki/FAQ#how-do-i-push-a-job-to-sidekiq-without-ruby
+
```

Comment:
@Mehonoshin I've updated the PR. What do you think about it?

Reply:
https://github.com/kevin-j-m/clockwork-test I'd use this to cover the clock.rb with specs file as well